gcloud/gcloud_kms.go

```diff
@@ -56,7 +55,7 @@ func New(
 	if !ok {
 		return nil, ErrInvalidKvdbProvided
 	}
-	ps := store.NewKvdbPersistenceStore(kv, kvdbPublicBasePath, kvdbDataBasePath)
+	ps := store.NewKvdbPersistenceStore(kv, kvdbPublicBasePath, "")
```

Comment:
Why are we removing the kvdbDataBasePath? It will break for existing users.

Reply:
I thought we don't store anything in the data path for ibm and gcloud. The data is encrypted and stored as the public part.
As of now only in case AWS we store the public part and encrypted data separately, right?